src/navigation/appNavigation.js

```diff
@@ -53,7 +53,8 @@ import SendTokenTransactionScreen from 'screens/SendToken/SendTokenTransaction';
 import SendCollectibleConfirmScreen from 'screens/SendCollectible/SendCollectibleConfirm';
 import PPNSendTokenAmountScreen from 'screens/Tank/SendToken/PPNSendTokenAmount';
 import HomeScreen from 'screens/Home';
-import MeScreen from 'screens/Me';
+import LoginScreen from 'screens/Home/Login';
+// import MeScreen from 'screens/Me';
```

Comment:
also commented?

Reply:
Yes, we need to decide if we're gonna drop that view or if we plan to link it from somewhere. Currently we are not using it, so I commented it out just to make it explicit. I could remove it but not sure :thinking: